heartbeat/autodiscover/builder/hints/monitors_test.go

```diff
@@ -203,7 +203,7 @@ func TestGenerateHints(t *testing.T) {
 			config: defaultConfig(),
 			logger: logp.NewLogger("hints.builder"),
 		}
-		cfgs := m.CreateConfig(test.event)
+		cfgs := m.CreateConfig(test.event, nil)
```

Comment:
I think you need to remove the `nil` here.

Reply:
👍